src/main/java/com/mailchimp/MailChimpErrorDecoder.java

```diff
@@ -8,17 +8,34 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.mailchimp.domain.MailChimpError;
 import feign.Response;
+import feign.RetryableException;
 import feign.codec.ErrorDecoder;
+
 import java.io.IOException;
+import java.sql.Date;
```

Comment:
java.sql.Date ? I also noticed that you are using java.util.Date directly instead of importing it.

Reply:
Oops, that was defiantly not intended, good catch.